media-plugins/qt-heif-image-plugin/metadata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>torokhov-s-a@yandex.ru</email>
+    <name>Sergey Torokhov</name>
+  </maintainer>
+  <maintainer type="project">
+    <email>qt@gentoo.org</email>
+    <name>Gentoo Qt Project</name>
+  </maintainer>
+  <longdescription>
+    This is a Qt image plugin for HEIF images, wrapping libheif.
+    It enables existing Qt applications to open and save .heic files.
+
+    Any application that (directly or indirectly) uses QImageReader
+    to open image files should automatically be able to use this plugin.
+    This has been successfully used with the following:
+    <pkg>media-gfx/lximage-qt</pkg>
+    <pkg>media-gfx/nomacs</pkg>
```

Comment:
not sure if I'd include this, it will eventually become outdated.

Reply:
Removed.